android/src/main/kotlin/com/appcues/flutter/FlutterElementTargetingStrategy.kt

```diff
@@ -0,0 +1,158 @@
+package com.appcues.flutter
+
+import android.app.Activity
+import android.graphics.Rect
+import android.os.Build
+import android.util.Size
+import android.view.View
+import android.view.ViewGroup
+import android.view.inspector.WindowInspector
+import androidx.core.graphics.Insets
+import androidx.core.view.ViewCompat
+import androidx.core.view.WindowInsetsCompat
+import com.appcues.ElementSelector
+import com.appcues.ElementTargetingStrategy
+import com.appcues.Screenshot
+import com.appcues.ViewElement
+import io.flutter.plugin.common.MethodCall
+import java.lang.reflect.Method
+
+internal class FlutterElementSelector(var identifier: String?): ElementSelector {
```

Comment:
Android side, but same on iOS - the new selector here has a single prop `"identifier"`

Reply:
If the value comes from a custom `AppcuesView` type, should we maybe use `appcuesID` like we do in the native SDKs instead of `identifier`? Realistically it doesn't matter one way or another. The only real upside I can conjure up is having  test experiences be slightly more portable across platforms.

Reply:
yep, I like that consistency, good call.

Reply:
updated in https://github.com/appcues/appcues-flutter-plugin/pull/43/commits/79b8770a9d66a30dbd3e879276209ac516120baf